### PR TITLE
satori readme

### PR DESCRIPTION
**Pull request by lugehorsam**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -200,9 +200,59 @@ doxygen
...
+
+You can also send arbitrary events to the server:
+
+```csharp
+
+await client.EventAsync(session, new Event("gameLaunched", DateTime.UtcNow));
+
+```
+
+This is only a subset of the Satori client API, so please see the documentation link listed earlier for the full API.
```
Comment (novabyte) [inline README.md:253]: I think we should add a link to this page: https://dotnet.docs.heroiclabs.com/html/namespace_satori.html

File: README.md [hunk 2]
```diff
@@ -200,9 +200,59 @@ doxygen
...
+
+```csharp
+var experiments = await client.GetExperimentsAsync(session);
+var flag = await client.GetFlagAsync(session, "FlagName");
+```
+
+You can also send arbitrary events to the server:
+
+```csharp
+
```
Comment (novabyte) [inline README.md:248]: nit: whitespace.

File: README.md [hunk 3]
```diff
@@ -200,9 +200,59 @@ doxygen
 
 # Satori
 
-This repository also contains the Satori client for use with the [Satori Liveops Server](https://heroiclabs.com/satori/).
+Satori is a liveops server for games that powers actionable analytics, A/B testing and remote configuration. Use the Satori .NET Client to coomunicate with Satori from within your .NET game.
```
Comment (novabyte) [inline README.md:203]: I think the whole Satori section of content should come before the **Contribute** section.

**Review by novabyte — APPROVED** [review]
@lugehorsam I've left a few small comments otherwise 👍

File: README.md (hunk 3 above)
Comment (lugehorsam) [reply README.md:203]: @novabyte good point, the section will be "nakama api" then "satori api", then "nakama contribute" and "satori contribute".

**Pull request merged**